modules/update-upgrading-oc-adm-upgrade-recommend.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * updating/updating_a_cluster/updating-cluster-cli.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="update-upgrading-oc-adm-upgrade-recommend_{context}"]
+= Finding recommended update paths with `oc adm upgrade recommend` (Technology Preview)
+
+When updating your cluster, it is useful to find out which versions are recommended for your update. While the `oc adm upgrade` command returns a list of the next available versions, this command decouples that information and narrows down your suggestions to recommend a new target release before you launch your update.
+
+The `oc adm upgrade recommend` command is read-only and will never alter any state in your cluster. 
+
+:FeatureName: The `oc adm upgrade recommend` command
+include::snippets/technology-preview.adoc[]
+
+[IMPORTANT]
+====
+While your cluster does not need to be a Technology Preview-enabled cluster, you must enable the `OC_ENABLE_CMD_UPGRADE_STATUS` Technology Preview environment variable, otherwise the {oc-first} will not recognize the command and you will not be able to use the feature. 
+====
+
+.Procedure 
+
+. Set the `OC_ENABLE_CMD_UPGRADE_STATUS` environmental variable to `true` by running the following command:
++
+[source,terminal]
+----
+$ export OC_ENABLE_CMD_UPGRADE_RECOMMEND=true
+----
+. Run the `oc adm upgrade recommend` command:
++
+[source,terminal]
+----
+$ oc adm upgrade recommend
+----
++
+.Example output for an update progressing successfully
```

Comment:
If we have to name this output, I'd call it something like "Example output when 4.y is recommended unconditionally, while the longest 4.(y+1) have assessed issues", or something, but that's a bit of a mouthful.  I don't think we should use "for an update progressing successfully", because we're thinking about kicking off an update, and that's mostly happening while the cluster is not already mid-update.  Since we only have the one example, we can maybe just say "Example output"?

Reply:
That's fine by me :)